src/internal.h

```diff
@@ -248,6 +248,7 @@ struct _GLFWinitconfig
         GLFWbool  menubar;
         GLFWbool  chdir;
     } ns;
+    GLFWbool      x11_xcb_vulkan_surface;
```

Comment:
Please use an anonymous struct `x11` and camel case like the sections above.

Reply:
Yes, of course. It should be OK.